.circleci/config/commands/go_test.yml

```diff
@@ -103,9 +106,17 @@ steps:
             TEST_DOCKER_NETWORK_ID=$(docker network create vaulttest)
           fi
 
+
+
           # Start a docker testcontainer to run the tests in
-          docker run -d -e TEST_DOCKER_NETWORK_ID \
-            -e DOCKER_CERT_PATH -e DOCKER_HOST -e DOCKER_MACHINE_NAME -e DOCKER_TLS_VERIFY -e NO_PROXY \
+          docker run -d \
+            -e TEST_DOCKER_NETWORK_ID \
+            -e GOPRIVATE \
```

Comment:
Feels like this one would fit better with the other "GO" variables in the docker exec below, but I don't suppose it matters.

Reply:
True.. For now I would like to merge as-is just to get these files to match, then changes can flow OSS->ENT like normal. If this gets confusing, we could try to isolate the ENT-specific portions and use an _ent.yml suffixed file for the non-OSS parts, but it seems like it's worth trying it this way round first as it's simpler.

Reply:
Addressed in https://github.com/hashicorp/vault/pull/11952